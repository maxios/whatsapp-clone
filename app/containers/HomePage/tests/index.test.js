import React from 'react';
import { shallow } from 'enzyme';
import AppBarComponent from '../../../components/AppBar';
import Menu from '../../../components/AppBar/MenuButton';

describe('AppBar Buttons', function() {
  const AppBar = shallow(<AppBarComponent Menu/>)

  it('should render Menu Button', function() {
    expect(AppBar).toMatchSnapshot();
  });

});


