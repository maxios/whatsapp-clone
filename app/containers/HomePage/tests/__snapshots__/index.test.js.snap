exports[`AppBar Buttons should render Menu Button 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <AppBar
        iconElementLeft={
            <Avatar
                size={40} />
        }
        iconElementRight={
            <div>
                <Menu />
                
                
                
            </div>
        }
        showMenuIconButton={true}
        style={
            Object {
                "background": "#eee",
                "height": "100%",
              }
        }
        title=""
        zDepth={1} />
</div>,
  "nodes": Array [
    <div>
      <AppBar
            iconElementLeft={
                  <Avatar
                        size={40} />
            }
            iconElementRight={
                  <div>
                        <Menu />
                        
                        
                        
                  </div>
            }
            showMenuIconButton={true}
            style={
                  Object {
                        "background": "#eee",
                        "height": "100%",
                      }
            }
            title=""
            zDepth={1} />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <AppBarComponent
        Menu={true} />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "Menu": true,
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <AppBar
                    iconElementLeft={
                              <Avatar
                                        size={40} />
                    }
                    iconElementRight={
                              <div>
                                        <Menu />
                                        
                                        
                                        
                              </div>
                    }
                    showMenuIconButton={true}
                    style={
                              Object {
                                        "background": "#eee",
                                        "height": "100%",
                                      }
                    }
                    title=""
                    zDepth={1} />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <AppBar
                    iconElementLeft={
                              <Avatar
                                        size={40} />
                    }
                    iconElementRight={
                              <div>
                                        <Menu />
                                        
                                        
                                        
                              </div>
                    }
                    showMenuIconButton={true}
                    style={
                              Object {
                                        "background": "#eee",
                                        "height": "100%",
                                      }
                    }
                    title=""
                    zDepth={1} />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <AppBarComponent
    Menu={true} />,
}
`;
